Gaijinizer patch/Troops.txt

```diff
@@ -15562,9 +15562,9 @@ captured by a bunch of faeries.#Triketos
 > CONTEXT: Troops/330/pages/1/16/Dialogue/enemy33
 > CONTEXT: Troops/331/pages/1/15/Dialogue/enemy33
 What's the matter?
-Did you run away because you didn't want to
-deal with the little guys?
-If you want, I can keep you company... ♡#DL
+Did you run away because you didn't want to play
+with those little guys?
+I can keep you company if you'd like... ♡#Strate
 > END STRING
```

Comment:
Line break between two and three.

Reply:
There's one more line above those three, I didn't change it so it didn't appear in the changes but all 4 lines are occupied for that text box.